inc/functions/cloudflare.php

```diff
@@ -41,15 +41,41 @@ function rocket_is_api_keys_valid_cloudflare( $cf_email, $cf_api_key, $cf_zone_i
 
 	try {
 		$cf_api_instance = new Cloudflare\Api( $cf_email, $cf_api_key );
-		$cf_user         = $cf_api_instance->get('user/');
-		$cf_zone         = $cf_api_instance->get('zones/'.$cf_zone_id);
+		$cf_user         = $cf_api_instance->get( 'user/' );
+		$cf_zone         = $cf_api_instance->get( 'zones/' . $cf_zone_id );
 
 		if ( ! isset( $cf_zone->success ) || empty( $cf_zone->success ) ) {
 			return new WP_Error( 'cloudflare_invalid_auth', $cf_zone->errors[0]->message );
 		}
 
 		if ( true === $cf_zone->success ) {
-		    return true;
+			$zone_found = false;
+
+			foreach ( $cf_zone->result as $zone ) {
+				if ( get_site_url() === $zone->name ) {
```

Comment:
I think `get_home_url()` is more appropriate here, the values between `get_home_url()` and `get_site_url()` can be different

Reply:
Please see what the official Cloudflare plugin uses.

Reply:
Cloudflare uses this function: https://www.screencast.com/t/3YHnvKTkqBE